src/features/calendar/components/CalendarWeekView/utils.ts

```diff
@@ -30,3 +30,9 @@ export function scrollToEarliestEvent(
 
   weekElement.scrollTo({ behavior: 'smooth', top: heightInPixels });
 }
+
+export function getDSTOffset(date: Date): number {
+  const jan = new Date(date.getFullYear(), 0, 1).getTimezoneOffset();
+  const jul = new Date(date.getFullYear(), 6, 1).getTimezoneOffset();
+  return (date.getTimezoneOffset() - Math.max(jan, jul)) / 60;
```

Comment:
Wow, I did not know that "the timezone offset is the offset [...], at the time represented by the Date object".

I had guessed that `Date` would only consider the current offset of the host system. Now I'm getting curious if this behaviour has unexpected consequences elsewhere.

Reply:
I wish JS had better date handling, especially time zones become very hacky. But this seems to work, provided daylight savings occurs sometime between January and June. Handling it manually would be a nightmare, so I'm glad this seems to work.